### PR TITLE
fix: stop process termination

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

File: src/ContentCreator.php [hunk 1]
```diff
@@ -718,6 +719,8 @@ protected function getRandomWeighted(array $list) {
    *   Flag to keep CSV files after import.
    */
   public function importCsvFiles($keep_content_files = FALSE) {
+    $this->storage->startContentCreationProcess();
```
Comment (alexpott) [inline src/ContentCreator.php:722]: This should only occur in the child processes. So after the fork.

File: src/ContentCreator.php [hunk 2]
```diff
@@ -782,6 +785,10 @@ public function importCsvFiles($keep_content_files = FALSE) {
             }
           }
 
+          if (PHP_SAPI === 'cli') {
```
Comment (alexpott) [inline src/ContentCreator.php]: This could be changed to a class exists to test if Drush is being used.

**Review by alexpott — CHANGES_REQUESTED** [review]

File: src/ContentCreator.php (hunk 1 above)
Comment (chrfritsch) [reply src/ContentCreator.php:722]: I did that first. But then the process is still failing

File: src/ContentCreator.php (hunk 2 above)
Comment (chrfritsch) [reply src/ContentCreator.php]: Yes, will do that

File: src/ContentCreator.php (hunk 1 above)
Comment (alexpott) [reply src/ContentCreator.php:722]: That's weird. How does it fail?

Comment (chrfritsch) [reply src/ContentCreator.php:722]: The terminate event still occurs twice at the end, which executes the RequestCloseSubscriber, which wants to clear the cache, which ends in a db lock.

Comment (alexpott) [reply src/ContentCreator.php:722]: You got to wonder why...

Does this happen even when you change the cache clear in the Drush command to drupal_flush_all_caches()?

Comment (chrfritsch) [reply src/ContentCreator.php:722]: With drupal_flush_all_caches I got even more problems. Then funnily the inContentCreationProcess var was never set in the child processes. Strange stuff.

**Review by alexpott — APPROVED** [review]

**Pull request merged**